webapp/nuxt.config.js

```diff
@@ -36,7 +38,8 @@ export default {
     // https://go.nuxtjs.dev/pwa
     '@nuxtjs/pwa',
```

Comment:
⭐ For Lighthouse reporting that your production website is installable as PWA (except HTTPS).